molecule/default/verify.yml

```diff
@@ -116,6 +116,18 @@
           - nix_hello_bin.stat.gid == nix_group.gid
           - nix_hello_bin.stat.mode == "0555"
 
+    - name: Read mount points
+      ansible.builtin.slurp:
+        src: "/proc/mounts"
+      register: proc_mounts
+
+    - name: Verify overlay mount for /nix
+      ansible.builtin.assert:
+        fail_msg: "/proc/mounts did not contain an overlay for /nix"
+        that:
+          - "{{ proc_mounts['content'] | b64decode \
+            | regex_search('overlay /nix') is not none }}"
```

Comment:
Why is this not `None`?

Reply:
This is what the [Ansible documentation](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#searching-strings-with-regular-expressions) suggests.

Reply:
I tried comparing against `None` and this actually fails.

Reply:
Ok, thanks for checking!